lib/json_api_resource/queryable.rb

```diff
@@ -45,7 +45,8 @@ def single_result(results)
 
         result.meta = results.meta
 
-        result.errors = ApiErrors(results.errors)
+        # There will never be errors here but this provides a good default for forms
+        result.errors = ActiveModel::Errors.new(result)
```

Comment:
since `result` is a `Resource` which has the `include ActiveModel::Validations` module, do we need this line?